pkg/storage/tsdb/config.go

```diff
@@ -23,12 +24,12 @@ var (
 
 // Config holds the config information for TSDB storage
 type Config struct {
-	Dir          string        `yaml:"dir"`
-	SyncDir      string        `yaml:"sync_dir"`
-	BlockRanges  DurationList  `yaml:"block_ranges_period"`
-	Retention    time.Duration `yaml:"retention_period"`
-	ShipInterval time.Duration `yaml:"ship_interval"`
-	Backend      string        `yaml:"backend"`
+	Dir            string            `yaml:"dir"`
+	BlockRanges    DurationList      `yaml:"block_ranges_period"`
+	Retention      time.Duration     `yaml:"retention_period"`
+	ShipInterval   time.Duration     `yaml:"ship_interval"`
+	Backend        string            `yaml:"backend"`
+	BucketStoreCfg BucketStoreConfig `yaml:"bucket_store_config"`
```

Comment:
I'm not sure the `Cfg` / `_config` suffix brings a real value here. I would simplify it to:

```
BucketStore BucketStoreConfig `yaml:"bucket_store"`
```